src/views/InteractiveMap/InteractiveMap.vue

```diff
@@ -300,7 +305,10 @@ export default Vue.extend({
         isSimulating: false,
         hasSimulationError: false,
         growthRate: null,
-        fluxes: null
+        fluxes: null,
+        // When adding a card from the interactive map, the dialog should be opened
+        // Method always exists on the default card
```

Comment:
I don't think we should rely on the method behaviour, it's not really related. Rather just add a new `withDialog` parameter to the `addCard` method.